djangocms_versioning/test_utils/extended_polls/migrations/0001_initial.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 4.1.3 on 2022-11-08 17:46
```

Comment:
I don't think that the migrations are actually required for test packages, maybe this is a side affect of the testing issue not using postgres. Doesn't really matter to be fair.